### PR TITLE
Updating turnOnCEC & turnOffCEC functions

### PR DESCRIPTION
**Pull request by krosamont**
I was not able to use these functions on my monitor. I have tried to adapt them.
you can try in the 2 following command lines:
/opt/vc/bin/tvservice -p && sudo chvt 6 && sudo chvt 7 # turn on the monitor
/opt/vc/bin/tvservice -o #turn off the monitor

### CODE REVIEW

File: MMM-CECControlNodeHelper.js [hunk 1]
```diff
@@ -63,14 +63,14 @@ exports.MMMCECControlNodeHelper = {
 
   turnOnCEC: function (callback) {
     this.execWrapper(
-      'echo "on 0" | cec-client ' + this.config.comport + ' -s -d 1',
+      '/opt/vc/bin/tvservice -p && sudo chvt 6 && sudo chvt 7',
```
Comment (nischi) [inline MMM-CECControlNodeHelper.js]: Is sudo needed? I ask because i think this will not work if you not run MagicMirror without Admin privilege.

**Review by nischi — COMMENTED** [review]
I have only one question, i'm a little bit concerned about the sudo

File: MMM-CECControlNodeHelper.js (hunk 1 above)
Comment (krosamont) [reply MMM-CECControlNodeHelper.js]: Sudo is not needed! Sorry I will remove it

**Comment by WaltmanBeta** [comment]
Sorry to put this here but this is the change that caused my MM to stop working when I finally did an update to modules this past week that I have been putting off for a while. Let me start by saying I am new to GitHub and programming raspberry pi so I am sorry if this is a noob issue. With the ￼changes from "cec-client" control to "tvservice" you essentially change the function of the command to disable the hdmi output not send the command to turn off the display. I have used this module for over a year with out issues until this change was pushed into master. I think this should be undone or an option to decide which method of control is to be used. Please help me if there is just an easy way to remedy (at least on my screen). Thanks

**Comment by krosamont** [comment]
If this change doesn't work for you. It means that there are some corrections to make.
Yes you're right. We should go back on the previous version and I will work on my side on the correction. Thx for the feed back

**Pull request merged**